### DISCUSSION TITLE
Question about the private MHDDoS script

### DISCUSSION THREAD

**Discussion by ProTechEx** [MHProDev/MHDDoS#322] [General]
Hi,

I just read on the Telegram 

buy Private DDoS script [MH VIP]
👇

https://vip.sellix.io/product/6251f743dbbf9

I am confused with this part:

🎩By: @m4s9b8
⚠️ Only layer4 amp methods working (x60)

🗡 @UnderFlood

Do I am understanding correct, that just "Only layer4 amp methods working (x60)" with this VIP script. As I remember, in the description was before the info something with 111 methods.

Could you guys explain more in details the benefits to free one ?

And how about a Webpanel? 
Is something planned or may currently in development?

Or does it will be never developed?

**Comment by fazlu3088** [comment]
Hello please define features of private DDoS?
